### PR TITLE
Update NPS definitions

### PR DESCRIPTION
**Pull request by esethna**
Update NPS metric definitions and include links to the new NPS dashboard:
https://mattermost.looker.com/dashboards/147

This PR introduces the definitions for "Quarterly Trailing NPS" and "NPS by Server Version" which are the primary methods of computing NPS used by the product team and represent the state of NPS for our customers.

New dashboard and metric definitions have been reviewed and approved by Lindsay and shared with PM team.

Note: Opening a new PR for these changes as the original PR was merged and reverted in error: https://github.com/mattermost/mattermost-handbook/pull/240

### CODE REVIEW

**Comment by esethna** [comment]
@enelson720 @rbradleyhaas @michaelschiffmm @lfbrock please help review and add your approval or comments before we merge. 

@enelson720 once merged we should update the references and chart data in any dashboards that display NPS to our [new NPS dashboard](https://mattermost.looker.com/dashboards/147), and sunset the [existing NPS dashboard](https://mattermost.looker.com/dashboards/41).

File: operations/business-operations/analytics/metrics-definitions.md
Comment (lfbrock) [inline operations/business-operations/analytics/metrics-definitions.md]: Let's leave out the google sheet here so we don't have to worry about keeping the links up to date

File: operations/business-operations/analytics/metrics-definitions.md
Comment (lfbrock) [inline operations/business-operations/analytics/metrics-definitions.md]: Same with these links - are we going to continue to use the same link over time? If not, consider either cutting or calling out that this is an example so the handbook is less likely to become dated.

**Review by lfbrock — DISMISSED** [review]

File: operations/business-operations/analytics/metrics-definitions.md
Comment (esethna) [reply operations/business-operations/analytics/metrics-definitions.md]: Sure, I'll remove them. Ideally I'd like to move these to productboard so these links are temporary

**Comment by esethna** [comment]
Thanks @lfbrock, updated

**Review by enelson720 — COMMENTED** [review]
LGTM

**Comment by esethna** [comment]
Removing review requirement from Michael as per his request, awaiting review from @rbradleyhaas

**Review by rbradleyhaas — APPROVED** [review]

**Pull request merged**
